docs/deploy/cloud.md

```diff
@@ -64,26 +64,46 @@ restate deployments register tunnel://example:9081
 
 To invoke services running on AWS Lambda, Restate Cloud needs to assume an AWS
 identity in the same account that the Lambda is deployed to. Create a new role
-that has permission to invoke your Lambdas and give it the following trust policy:
+that has permission to invoke your Lambda handlers, and give it the following
+trust policy.
 
 ```json
 {
-  "Sid": "AllowRestateCloudToAssumeRole",
-  "Effect": "Allow",
-  "Principal": {
-    "AWS": "arn:aws:iam::654654156625:role/RestateCloud"
-  },
-  "Action": ["sts:AssumeRole", "sts:TagSession"],
-  "Condition": {
-    "StringEquals": {
-      "sts:ExternalId": "$ENVIRONMENT_ID"
-    }
-  }
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::654654156625:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "aws:PrincipalArn": "arn:aws:iam::654654156625:role/RestateCloud",
+                    "sts:ExternalId": "${ENVIRONMENT_ID}"
+                }
+            }
+        },
+        {
+            "Sid": "AllowTagSession",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::654654156625:root"
+            },
+            "Action": "sts:TagSession",
+            "Condition": {
+                "StringEquals": {
+                    "aws:PrincipalArn": "arn:aws:iam::654654156625:role/RestateCloud"
+                }
+            }
```

Comment:
Technically we can remove this and shorten the policy to just:

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::654654156625:root"
            },
            "Action": "sts:AssumeRole",
            "Condition": {
                "StringEquals": {
                    "aws:PrincipalArn": "arn:aws:iam::654654156625:role/RestateCloud",
                    "sts:ExternalId": "${ENVIRONMENT_ID}"
                }
            }
        },
        {
            "Sid": "AllowTagSession",
            "Effect": "Allow",
            "Principal": {
                "AWS": "arn:aws:iam::654654156625:root"
            },
            "Action": "sts:TagSession"
        }
    ]
}
```

It's no less secure since the role principal is the only one that can `AssumeRole` - it's shorter but less pedantic :-)